src/applications/personalization/profile360/tests/components/PaymentInformationEditModal.unit.spec.jsx

```diff
@@ -54,20 +54,18 @@ describe('<PaymentInformationEditModal/>', () => {
   it('submits', () => {
     const onSubmit = sinon.spy();
 
-    let props = set('fields.accountNumber.field.value', '123456', defaultProps);
-    props = set(
-      'fields.financialInstitutionRoutingNumber.field.value',
-      '123456789',
-      props,
-    );
-    props = set('onSubmit', onSubmit, props);
+    const props = set('onSubmit', onSubmit, defaultProps);
 
-    const wrapper = shallow(<PaymentInformationEditModal {...props} />);
-    const event = {
-      preventDefault() {},
-    };
+    const wrapper = mount(<PaymentInformationEditModal {...props} />);
```

Comment:
Had to switch from a shallow render to a full render or the `form` element would never be rendered since that is now a byproduct of the `SchemaForm` from rendering. Honestly these two tests can probably be removed or radically changed. They were originally just checking to make sure that the custom validation logic worked. But since we are now using a well tested `SchemaForm` component that handles the validation, we probably just need to make sure that a `SchemaForm` is configured and rendered.